react-library/CHANGELOG.md

```diff
@@ -0,0 +1,68 @@
+### Changelog
+
+All notable changes to this project will be documented in this file. Dates are displayed in UTC.
+
+Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+
+#### [0.0.6](https://github.com/justifi-tech/web-component-library/compare/0.0.5...0.0.6)
```

Comment:
What about all of the versions after this? I believe we are on version `0.0.15`. A lot of the changes shown below from `0.0.1` are actually changes that happened recently, after `0.0.6`